patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -0,0 +1,166 @@
+												<?php echo esc_html_x( 'Shop at Audible', 'Hero - Overlapped book cover pattern button 3', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+										<!-- wp:button {"backgroundColor":"contrast","textColor":"base","width":100,"className":"is-style-outline","style":{"border":{"radius":"50px"},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"1rem","bottom":"1rem"}}}} -->
+										<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size is-style-outline" style="font-size:1rem">
+											<a class="wp-block-button__link has-base-color has-contrast-background-color has-text-color has-background has-link-color wp-element-button" href="#" style="border-radius:50px;padding-top:1rem;padding-bottom:1rem">
+												<?php echo esc_html_x( 'Shop at Barnes & Noble', 'Hero - Overlapped book cover pattern button 4', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+									</div>
+									<!-- /wp:buttons -->
+
+								</div>
+								<!-- /wp:column -->
+
+							</div>
+							<!-- /wp:columns -->
+
+							<!-- wp:spacer {"height":"40px","style":{"layout":{"selfStretch":"fixed","flexSize":"3rem"}}} -->
+							<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+							<!-- /wp:spacer -->
+
+							<!-- wp:paragraph {"metadata":{"name":"BottomText"},"style":{"typography":{"fontSize":"1.13rem","fontStyle":"normal","fontWeight":"500"}}} -->
+							<p style="font-size:1.13rem;font-style:normal;font-weight:500">
+								<?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>.', 'Hero - Overlapped book cover pattern bottom text', 'twentytwentyfive' ) ); ?>
+							</p>
+							<!-- /wp:paragraph -->
+						</div>
+						<!-- /wp:group -->
+
+					</div>
+					<!-- /wp:group -->
+
+				</div>
+				<!-- /wp:column -->
+
+				<!-- wp:column {"verticalAlignment":"top","width":"45%"} -->
+				<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:45%">
+
+					<!-- wp:image {"aspectRatio":"3/4","scale":"cover","sizeSlug":"full","linkDestination":"none","align":"center"} -->
+					<figure class="wp-block-image aligncenter size-full">
+						<img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/book-image.webp" alt="<?php echo esc_attr__( 'Book Image', 'twentytwentyfive' ); ?>" style="aspect-ratio:3/4;object-fit:cover"/>
+					</figure>
+					<!-- /wp:image -->
+
+				</div>
+				<!-- /wp:column -->
+
+			</div>
+			<!-- /wp:columns -->
+
+		</div>
+		<!-- /wp:group -->
+
+	</div>
+</div>
+<!-- /wp:cover -->
+
+
+
+
+
```

Comment:
Please only leave one single blank line at the end of the file.

Reply:
Blank lines removed.